dev/managedservicesplatform/stacks/cloudrun/cloudrun.go

```diff
@@ -66,6 +66,7 @@ type Variables struct {
 	IAM iam.CrossStackOutput
 
 	Service     spec.ServiceSpec
+	Repository  string
```

Comment:
```suggestion
    // Repository is the source code repository for the image to deploy
	Repository  string
```

Repository is ambiguous in the context of images, as many image hosts also call themselves "repositories" - a clarifying docstring here would be good I think, just to be safe